system/inc/system_sleep_configuration.h

```diff
@@ -88,6 +88,9 @@ class SystemSleepConfigurationHelper {
     }
 
     EnumFlags<SystemSleepNetworkFlag> networkFlags(network_interface_index index) const {
+        if (config_->version < HAL_SLEEP_VERSION) {
```

Comment:
Needs to be a fixed version when this feature appeared in, otherwise once we bump it again it will fail again.

Reply:
Good catch! 👍